src/handlers/transfers/handler.js

```diff
@@ -410,71 +407,37 @@ const getTransfer = async (error, messages) => {
       histTimerEnd({ success: false, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
       return true
     }
-    if (!await Validator.validateParticipantByName(message.value.from)) {
-      Logger.info('TransferService::getTransferHandler::participantCheck::doesntExist:: FSP Id does not exist')
-      if (!Kafka.Consumer.isConsumerAutoCommitEnabled(kafkaTopic)) {
-        await consumer.commitMessageSync(message)
-      }
-      histTimerEnd({ success: true, fspId: Config.INSTRUMENTATION_METRICS_LABELS.fspId })
-      return true
-    }
+    const AL = 'G'
```

Comment:
For readability please rename to actionLetter also i would store 'G' under a constant or enum with a name that explains what it is

Reply:
ok